tests/test_api.py

```diff
@@ -69,3 +69,9 @@ def test_pytket_classical_only(self) -> None:
             "returns": [["a", 0], ["b", 0]],
             "args": [["b", 2], ["a", 1]],
         }
+
+    def test_qasm_to_phir(self) -> None:
+        """Test the qasm string entrypoint works."""
+        qasm = 'OPENQASM 2.0;include "qelib1.inc";qreg q[3];h q; ZZ q[1],q[0];creg cr[3];measure q[0]->cr[0];measure q[1]->cr[0];'  # noqa: E501
```

Comment:
Will be easier to read if it's a multiline string.

Reply:
Yeah, fair. Updated.